homeassistant/components/huawei_lte/quality_scale.yaml

```diff
@@ -0,0 +1,76 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
`test_urlize_plain_host` is testing that `user_input` is mutated and I think that is a bad practice. I would rather see that we call the mock correctly with the right URL

Reply:
Noted in 6f4b51d1eebeb8405feeff141d01bde7000425b1